### ISSUE TITLE
No preview window and phonetic does not work

### ISSUE THREAD

**Issue by moshiur05** [OpenBangla/OpenBangla-Keyboard#150]
Hello i'm using openbangla keyboard on arch linux, the aur repo does not work, shows build error. And anything after #130 builds properly but phonetic does not work and no preview window is shown.

**Comment by mominul** [comment]
Hi!
It'd be best to use the versions [we have released](https://github.com/OpenBangla/OpenBangla-Keyboard/releases/tag/1.5.1). Currently on going developments do happen on the master branch which is the norm, so any kind of bug is supposed to be there. As we haven't performed enough tests to release it. So I'd recommend you to install the latest version we released so far.
It's unfortunate that I cannot test and fix the bug currently as my PC is currently broken.

**Comment by moshiur05** [comment]
No worries. I'm currently using #134 . It's working fine so far. Thanks for the amazing software.

**Comment by bdeshi** [comment]
> Hello i'm using openbangla keyboard on arch linux, the aur repo does not work, shows build error. ...

@moshiur05 which aur package are you using? I'm interested in those build errors if you were trying this one: https://aur.archlinux.org/packages/openbangla-keyboard/

**Comment by moshiur05** [comment]
Yes, I used the aur package. I'm currently using pacaur to manage aur. Here are the build errors :
`-- Build files have been written to: /home/moshiur/.cache/pacaur/openbangla-keyboard/src/OpenBangla-Keyboard-1.5.1/build
Scanning dependencies of target libShared
[  2%] Building CXX object src/shared/CMakeFiles/libShared.dir/Settings.cpp.o
[  5%] Linking CXX static library liblibShared.a
[  5%] Built target libShared
Scanning dependencies of target libEngine
[  8%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/Layout.cpp.o
[ 11%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/MethodPhonetic.cpp.o
[ 13%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/MethodFixedLayoutModern.cpp.o
[ 16%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/phoneticparser.cpp.o
[ 19%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/FixedLayoutParser.cpp.o
[ 22%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/regexparser.cpp.o
[ 25%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/AutoCorrect.cpp.o
[ 27%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/Database.cpp.o
[ 30%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/cachemanager.cpp.o
[ 33%] Building CXX object src/engine/libengine/CMakeFiles/libEngine.dir/PhoneticSuggestion.cpp.o
[ 36%] Linking CXX static library liblibEngine.a
[ 36%] Built target libEngine
Scanning dependencies of target OpenBangla-Engine
[ 38%] Building CXX object src/engine/ibus/CMakeFiles/OpenBangla-Engine.dir/ibus_engine.cpp.o
In file included from /home/moshiur/.cache/pacaur/openbangla-keyboard/src/OpenBangla-Keyboard-1.5.1/src/engine/ibus/ibus_engine.cpp:22:
/home/moshiur/.cache/pacaur/openbangla-keyboard/src/OpenBangla-Keyboard-1.5.1/src/engine/ibus/ibus_keycode.h: In function ‘int ibus_keycode(guint)’:
/home/moshiur/.cache/pacaur/openbangla-keyboard/src/OpenBangla-Keyboard-1.5.1/src/engine/ibus/ibus_keycode.h:193:15: error: ‘out_of_range’ in namespace ‘std’ does not name a type
  193 |   catch (std::out_of_range) {
      |               ^~~~~~~~~~~~
make[2]: *** [src/engine/ibus/CMakeFiles/OpenBangla-Engine.dir/build.make:83: src/engine/ibus/CMakeFiles/OpenBangla-Engine.dir/ibus_engine.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:246: src/engine/ibus/CMakeFiles/OpenBangla-Engine.dir/all] Error 2
make: *** [Makefile:172: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
:: cmake is now an orphan package
:: failed to build openbangla-keyboard package(s)`

**Comment by bdeshi** [comment]
```
/home/moshiur/.cache/pacaur/openbangla-keyboard/src/OpenBangla-Keyboard-1.5.1/src/engine/ibus/ibus_keycode.h:193:15: error: ‘out_of_range’ in namespace ‘std’ does not name a type
  193 |   catch (std::out_of_range) {
      |               ^~~~~~~~~~~~
```
@mominul any suggestions?
